client/app/core/core-module.runner.js

```diff
@@ -36,8 +40,18 @@ export function CoreModuleRunner(
         .then(() => {
             appHealthService.changeHealthyStatus(true);
 
-            getVersion();
-            updateCompanyLogo();
+            Promise.allSettled([
+                getVersion(),
+                updateCompanyLogo(),
+            ]).then(() => {
+                $ngRedux.dispatch(setApplicationConfig({
+                    api: API_URL,
```

Comment:
I think we should add somewhere a comment (TODO), that we should use `API_URL` only for initial calls, and then use tenant-specific one which we can get from BE